index_en.html

```diff
@@ -235,16 +235,10 @@ <h2 class="clearfix">
 								</a>
 							</td>
 						</tr>
-						<!--
 						<tr>
-							<th>参加者数</th>
-							<td>300 名</td>
+							<th>Ticket</th>
+							<td>Need a ticket.Please buy from <a href='https://scalaconfjp.doorkeeper.jp/events/53530'>here</a></td>
```

Comment:
Please purchase the tickets from <a href='https://scalaconfjp.doorkeeper.jp/events/53530'>Doorkeeper</a>.